src/session-pool.ts

```diff
@@ -458,28 +433,12 @@ export class SessionPool extends EventEmitter implements SessionPoolInterface {
     return this._pending + this._pendingPrepare;
   }
 
-  /**
-   * Current number of waiters for a read-only session.
-   * @type {number}
-   */
-  get numReadWaiters(): number {
```

Comment:
Same case as `currentWriteFraction` , `numReadWaiters` and `numWriteWaiters` are public methods and removing these will also be a breaking change. 
We can mark these methods as deprecated and probably return 0 ?

Reply:
ah.. yes, thanks! done